### PR TITLE
linted state, databases and api

### PR DESCRIPTION
**Pull request by antonlerner**
linted state, databases and api

### CODE REVIEW

File: api/node.go [hunk 1]
```diff
@@ -20,31 +22,37 @@ type StateAPI interface {
 	Exist(address types.Address) bool
 }
 
+// NetworkAPI is an API to nods gossip network
```
Comment (y0sher) [inline api/node.go]:
```suggestion
// NetworkAPI is an API to nodes gossip network
```

File: database/memory_database_iterator.go [hunk 2]
```diff
@@ -48,6 +56,7 @@ func (iter *MemDatabaseIterator) Last() bool {
 	return true
 }
 
+// Prev moves the iterator one item bacj
```
Comment (y0sher) [inline database/memory_database_iterator.go]:
```suggestion
// Prev moves the iterator one item back
```

**Review by y0sher — APPROVED** [review]
I've noticed most of your comments do not end with a period `.`
if I recall correctly the go doc format requires it to end with a period `.`,
 sorry :/

**Comment by antonlerner** [comment]
bors merge

**Comment by antonlerner** [comment]
bors merge

**Comment by antonlerner** [comment]
bors merge

**Comment by noamnelke** [comment]
Merged as part of **Consolidated linting (#1958)**

**Pull request closed without merging**